README.md

```diff
@@ -1,13 +1,184 @@
-# Kin core Stellar SDK for Android
+# Kin core SDK for Android
 Android library responsible for creating a new Stellar account and managing KIN balance and transactions.
 ![Kin Token](kin_android.png)
 
-## Disclaimer
+## Build
+
+* Add this to your module's `build.gradle` file.
+```gradle
+repositories {
+    ...
+    maven {
+        url 'https://jitpack.io'
+    }
+}
+...
+dependencies {
+    ...
+
+    compile "com.github.kinfoundation:kin-core-android:<latest version>"
+}
+```
+
+## Usage
+### Connecting to a service provider
+Create a new `KinClient` with two arguments: an android `Context` and a `ServiceProvider`. 
+
+A `ServiceProvider` provides details of how to access the Stellar horizon end point.
+The example below creates a `ServiceProvider` that will be used to connect to the main (production) Stellar 
+network
+```java
+ServiceProvider infuraProvider =  
+    new ServiceProvider("https://horizon.stellar.org", ServiceProvider.NETWORK_ID_MAIN));
```

Comment:
it's specific to Horizon, maybe call the class `HorizonServiceProvider`?

Reply:
it's not specific to horizon, the same API was used with ethereum, it's general blockchain endpoint parameters